### PR TITLE
Giving tuesday updates

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds category header widget area functionality like we did for MWCIR at https://github.com/INN/umbrella-mwcir/pull/43
- Shows the donate button in the header on mobile
- Updates styling for Largo donate + subscribe button in the footer
Screen Shot 2020-05-04 at 3 44 22 PM

## Why

For #39

## Testing/Questions

Features that this PR affects:

- Donate button in mobile menu
- Category archive widget area in header
- Footer widget styling

Questions that need to be answered before merging:

- [ ] Is this PR targeting the correct branch in this repository?

Steps to test this PR:

Donate button:
1. View the site on mobile and verify the donate button appears in the header

Category header widget area:
1. Test the category archive widget area the same way we did for MWCIR https://github.com/INN/umbrella-mwcir/pull/43

Footer widgets:
1. Add a Largo Donate widget to the Footer 2 widget area
2. Add an HTML widget under the Donate widget with this text
```
<a class="btn btn-subscribe" href="/subscribe/">Subscribe</a>
```
3. View on the frontend and verify it looks ok on desktop and mobile
3.

### CODE REVIEW

File: wp-content/themes/theme-publicsource/functions.php [hunk 1]
```diff
@@ -144,3 +144,23 @@ function publicsource_widget_custom_fields_form( $widget, $args, $instance ) {
...
 	add_action( 'in_widget_form', 'publicsource_widget_custom_fields_form', 1, 3 );
 });
+
+/**
+ * Adds the ability to include a widget area in the header of a category archive page.
+ * The widget area must use the same name as the slug of the category it is going to be used on.
+ * 
+ * If a widget area with a matching name does not exist or is empty, no widget area will be displayed.
+ * 
+ * @see https://github.com/INN/umbrella-mwcir/issues/42
```
Comment (benlk) [inline wp-content/themes/theme-publicsource/functions.php:154]: Might want to add a link to this PR as well.

File: wp-content/themes/theme-publicsource/css/style.css [hunk 2]
```diff
@@ -666,18 +666,31 @@ body.normal.page.page-template-page-clean article.page > header h1 {
...
+  margin-top: 1em;
+}
+#site-footer #supplementary .btn.btn-subscribe {
+  background: 0 0;
+  color: #418cff;
+  text-transform: uppercase;
+  font-family: 'tk-mallory-n7', sans-serif;
+  font-size: 2em;
+  width: 100%;
+}
```
Comment (benlk) [inline wp-content/themes/theme-publicsource/css/style.css:693]: Can we get hover colors for these buttons?

File: wp-content/themes/theme-publicsource/css/style.css [hunk 3]
```diff
@@ -666,18 +666,31 @@ body.normal.page.page-template-page-clean article.page > header h1 {
...
-  #header-extras li.donate {
-    display: none;
-  }
 }
 @media (min-width: 391px) {
-  .nav-shelf li.donate {
-    display: none;
+  #header-extras li.donate {
+    display: block;
   }
```
Comment (benlk) [inline wp-content/themes/theme-publicsource/css/style.css:673]: Are we okay with it appearing twice when the menu is opened?

Screen Shot 2020-05-04 at 16 18 47

Comment (joshdarby) [reply wp-content/themes/theme-publicsource/css/style.css:673]: From @MirandaEcho in slack:

> yes to being in both, that's fine

File: wp-content/themes/theme-publicsource/css/style.css (hunk 2 above)
Comment (joshdarby) [reply wp-content/themes/theme-publicsource/css/style.css:693]: Also from @MirandaEcho in Slack:

> no hover color is fine for now

File: wp-content/themes/theme-publicsource/functions.php (hunk 1 above)
Comment (joshdarby) [reply wp-content/themes/theme-publicsource/functions.php:154]: Added in https://github.com/INN/umbrella-publicsource/pull/40/commits/e58c234db90ef2eff1a09ef161f70c0f12860ce1

**Pull request merged**